app/views/admin/snap_applications/index.html.erb

```diff
@@ -58,6 +58,26 @@ It renders the `_table` partial to display details about the resources.
   <%= render "funnel_stage", stage: FunnelStage.new(previous_cohort: all_resources, cohort: all_resources.emailed_client, name: "Emailed Client") %>
 </section>
 
+<section class="main-content__snap-application-legend">
+  <strong>Legend:</strong>
+
+  <span class="drive_success">
+    Successful Drive
+  </span>
+
+  <span class="drive_in_flight">
+    Driving In Progress
```

Comment:
seems like we don't have anything to put in this column for now -- remove or update logic to have in flight be a thing?

Reply:
I'll remove this for now bc, yeah - if it's not a thing, it shouldn't be here!